plugins/outputs/logzio/logzio_test.go

```diff
@@ -0,0 +1,35 @@
+package logzio
+
+import (
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestNewLogzioOutput(t *testing.T) {
+	l := CreateDefultLogizoOutput()
+	require.Equal(t, l.CheckDiskSpace, defaultLogzioCheckDiskSpace)
+	require.Equal(t, l.DiskThreshold, defaultLogzioDiskThreshold)
+	require.Equal(t, l.DrainDuration, defaultLogzioDrainDuration)
+	require.Equal(t, l.URL, defaultLogzioURL)
+
+	require.Equal(t, l.Token, "")
+}
+
+func TestConnectAndWrite(t *testing.T) {
+	l := CreateDefultLogizoOutput()
+	l.Token = "123456789"
+
+	err := l.Connect()
```

Comment:
would be best if it didn't actually connect to a server here and instead mocked out an http server and tested the format.

Reply:
@ssoroka I think this can be problematic since you need to have an account at logz.io to test it with real server

Reply:
just this one remaining item to get it merged. :)

Reply:
@ssoroka thank you! I changed the logic a bit. The sending portion is tested in the sender so instead of testing it here I've extracted the parsing part and tested that it's valid